packages/react/src/components/participant/ParticipantTile.tsx

```diff
@@ -148,7 +151,7 @@ export function ParticipantTile({
             </div>
           </>
         )}
-        <FocusToggle trackSource={trackRef.source} />
+        <FocusToggle trackRef={trackRef} />
```

Comment:
only wondering about naming here of `trackRef`. 

`track` is ambiguous, but are we using `trackRef` as a property somewhere else already?